### ISSUE TITLE
[Task] Integration test to my emulator

### ISSUE THREAD

**Issue by rotekii** [element-hq/element-x-android#2819]
Hi , I'm trying to follow the steps for the [integration tests ](https://github.com/element-hq/element-x-android/blob/develop/docs/integration_tests.md), the 3 instances are working fine.

Now that it's done i still need to sliding sync, but i found it hard to follow the instructions, is there any easier way to do so?

**Comment by spaetz** [comment]
Given this is a request for sliding sync documentation, can this issue be closed?
